Products/CMFQuickInstallerTool/utils.py

```diff
@@ -22,6 +22,8 @@
     'ZODBMountPoint', 'ZReST', 'ZSQLMethods',
 ])
 
+UNINSTALL_ID = 'uninstalled'
```

Comment:
@pbauer Any reason that UINSTALL_ID is declared here as well as in QuickInstallerTool.py?